seg_utils/src/label_main.py

```diff
@@ -202,31 +251,32 @@ def initLabels(self):
         self.current_labels = [Shape(image_size=self.image_size, label_dict=_label,
                                      color=self.getColorForLabel(_label['label']))
                                for _label in labels]
-        self.polyList.updateList(self.current_labels)
+        self.polyFrame.polyList.updateList(self.current_labels)
+        self.polyFrame.commentList.initComments(self.current_labels)
 
     def initImage(self):
```

Comment:
This function should be inside ImageViewer and the file to be loaded should be sent with a signal.

Reply:
This can be done in a different pull request later